### PR TITLE
Cds 93147

### PR DESCRIPTION
**Pull request by roshnisarangadharan**

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-deployment-faqs.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+This limitation is imposed by [Helm](https://helm.sh/docs/chart_template_guide/getting_started/#adding-a-simple-template-call).
+
+To resolve this issue, please ensure that your release name falls within the 53 character range.
+
+You can achieve this by using the following expression in **Release Name** to shorten the release name: `<+<+INFRA_KEY>.substring(0,7)>`.
+
+### Troubleshoot 401 Not Authorized issue with Helm connector
+
+First, ensure the credentials that you have passed to the Helm HTTP or OCI Connector are still valid.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-deployment-faqs.md:67]:
```suggestion
First, ensure the credentials that you have passed to the Helm HTTP or OCI connector are still valid.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart.md [hunk 2]
```diff
@@ -77,82 +62,86 @@ For a detailed explanation of Helm deployments, go to [Deploy Helm charts](/docs
...
 
-- Rollback: Harness does not perform rollback. Instead, Harness uses Helm's native rollback functionality. This is great for quickly adopting based of your existing setup.
+- Rollback: Harness does not perform rollback. Instead, Harness uses Helm's native rollback functionality. This approach works well if you want to use your existing setup.
 - Harness will honor the user's pre and post install hooks configured in the Helm chart.
 
 ### Cons
 
 - Versioning: Native Helm does not version deployments.
-- No progressive deployment support, no blue green or canary deployment types are supported (coming soon).
+- No progressive deployment support. Also, Blue Green or Canary deployment types are not supported (coming soon).
```
Comment (SushrutHarness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart.md:133]:
```suggestion
- No progressive deployment support. Also, Blue/Green or Canary deployment types are not supported (coming soon).
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart.md [hunk 3]
```diff
@@ -178,68 +167,10 @@ If you select to build your own delegate and include only those tools needed for
...
-The "invalid release name" error is due to the length of the release name exceeding the maximum limit of 53 characters.
-
-This limitation is imposed by [Helm](https://helm.sh/docs/chart_template_guide/getting_started/#adding-a-simple-template-call).
-
-To resolve this issue, please ensure that your release name falls within the 53 character range.
+## See also
 
-You can achieve this by using the following expression in **Release Name** to shorten the release name: `<+<+INFRA_KEY>.substring(0,7)>`.
+* [CD pipeline basics](/docs/continuous-delivery/get-started/key-concepts)
+* [Deploy using Helm Chart](/docs/continuous-delivery/get-started/cd-tutorials/helm-chart)
```
Comment (SushrutHarness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart.md:173]: Maybe add the helm-deployment-faqs to this See also section?

**Review by SushrutHarness — APPROVED** [review]
few suggestions, but LGTM

**Review by krishi0408 — APPROVED** [review]

**Pull request merged**